components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -2437,4 +2447,160 @@ public void testModifyGroupEdit(boolean isPrivileged,
             }
         return testCases.toArray(new Object[testCases.size()][]);
     }
+
+    /**
+     * Tests if an enumeration not used can be deleted.
+     * This should pass but it can be restored using the method
+     * <code>resetEnumerations</code>.
+     * @throws Exception
+     */
+    @Test
+    public void testDeleteEnumerationsbyRestrictedSystemUser() throws Exception {
+        final IAdminPrx svc = root.getSession().getAdminService();
+        final List<AdminPrivilege> expectedPrivileges = new ArrayList<AdminPrivilege>();
+        final String lightAdminName = UUID.randomUUID().toString();
+        final String lightAdminPassword = UUID.randomUUID().toString();
```

Comment:
Would using an existing method like `loginNewAdmin` make this setup easier?

Reply:
I started to write more tests that I initially anticipated. :-)
I will refactor that part since it starts to be too much "copy/paste"